src/OpenLoco/Windows/PromptBrowseWindow.cpp

```diff
@@ -113,7 +114,7 @@ namespace OpenLoco::Ui::Windows::PromptBrowse
         Utility::strcpy_safe(_filter, filter);
 
         changeDirectory(directory.make_preferred());
-        inputSession = Ui::TextInput::InputSession(baseName, 200);
+        inputSession = Ui::TextInput::InputSession(baseName, PATH_MAX);
```

Comment:
Please revert PATH_MAX will be far too large and crash all sorts of things if it ever tries to copy the value into the internal buffers.

Reply:
Saw this too late. What size would you recommend? I think 31 chars is too short, at any rate.